src/FSharp.Build/FSharpEmbedResXSource.fs

```diff
@@ -10,14 +10,20 @@ open System.Xml.Linq
 open Microsoft.Build.Framework
 open Microsoft.Build.Utilities
 
-type FSharpEmbedResXSource() =
-    let mutable _buildEngine: IBuildEngine MaybeNull = null
-    let mutable _hostObject: ITaskHost MaybeNull = null
+type FSharpEmbedResXSource() as this =
+    inherit Task()
```

Comment:
Mhm I am not convinced why this is a better way to do things here. Probably not a big deal but just pointing out that there are currently four modules in this project that implement ITask and only one that inherits Task. I'd rather prefer consistency (to whichever side), probably in some follow-up.

Reply:
`Task` gives us access to the `Log` object. `ITask` is an advanced interface; there's no reason to manually implement it.